src/app/space/forge-wizard/import-wizard.component.html

```diff
@@ -0,0 +1,58 @@
+<forge-exception *ngIf="error" [error]="error"
+                 (onCancel)="cancel($event)">
+</forge-exception>
+<pfng-wizard *ngIf="!error" #wizard
+             [config]="config"
+             (onCancel)="cancel($event)"
+             (onNext)="nextClicked($event)"
+             (onPrevious)="previousClicked($event)"
+             (onStepChange)="stepChanged($event)">
+
+  <div style="min-height: 450px" class="wizard-pf-loading blank-slate-pf loading" *ngIf="isLoading">
```

Comment:
Looks like this div should be inside the .wizard-pf-main so that it appears properly centered in the content area of the wizard. (then hopefully no inline min-height necessary)

Reply:
@srambach looking with @edewit we found https://github.com/patternfly/patternfly-ng/blob/master/src/app/wizard/wizard.component.html#L29 maybe a better option is to reuse the loading screen from the patternfly wizard itself.